Mobile/stockdogmobile/screens/ideaFeed.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { TouchableOpacity, View, Text } from 'react-native';
+import styles from '../style/screens/feed';
+
+export default class IdeaFeed extends Component {
+
+   constructor(props) {
```

Comment:
Unnecessary constructor

Reply:
thank you for this thoughtful comment

Reply:
not every hero wears a cape
![image](https://user-images.githubusercontent.com/15795866/49274079-f706f700-f42b-11e8-8167-39d83952f7fd.png)